### PR TITLE
Update Traditional Chinese (zh_tw.json)

### PR DESCRIPTION
**Pull request by yichifauzi**
Update and Fix up Traditional Chinese localization

### CODE REVIEW

File: src/main/resources/assets/viafabricplus/lang/zh_tw.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "base.viafabricplus.online_mode": "線上模式",
   "base.viafabricplus.copy_link": "複製連結",
   "base.viafabricplus.something_went_wrong": "發生錯誤！請稍後再試",
-  "base.viafabricplus.via_translates_to": "轉換為：%s",
-  "base.viafabricplus.server_version": "伺服器版本：%s",
+  "base.viafabricplus.via_translates_to": "轉換為： %s",
```
Comment (FlorianMichael) [inline src/main/resources/assets/viafabricplus/lang/zh_tw.json:17]: why does this add a new space? same below

**Review by FlorianMichael — APPROVED** [review]
Thank you, if you care to join my discord server you can get pinged once this file needs updating.
https://discord.com/invite/97GXQxuf7W

**Comment by yichifauzi** [comment]
> Thank you, if you care to join my discord server you can get pinged once this file needs updating. https://discord.com/invite/97GXQxuf7W

I already joined Discord server, Thank you Developer
For merge

**Comment by FlorianMichael** [comment]
what's your discord name? So you can get the translator role

**Pull request merged**